lua/autorun/server/sv_model_regulator.lua

```diff
@@ -9,7 +9,7 @@ local entityMeta = FindMetaTable("Entity")
 local setModel = entityMeta.SetModel
 
 function playerMeta:SetModel(desiredModel)
-	if modelIsProhibited[desiredModel] then return setModel(self, defaultModel) end
+	local model = modelIsProhibited[desiredModel] and defaultModel or desiredModel end
```

Comment:
```suggestion
local model = ( modelIsProhibited[desiredModel] and defaultModel ) or desiredModel end
```

you madman

Reply:
this just the way LUA ternaries
http://lua-users.org/wiki/TernaryOperator

Reply:
I just think it's clearer with the parens, that's all. Should still function the same.